react/features/connection-indicator/statsEmitter.js

```diff
@@ -33,27 +41,39 @@ const statsEmitter = {
 
         conference.on(JitsiConnectionQualityEvents.REMOTE_STATS_UPDATED,
             (id, stats) => this._emitStatsUpdate(id, stats));
+
+        conference.on(
+            JitsiE2ePingEvents.E2E_RTT_CHANGED,
+            (participant, e2eRtt) => this._emitE2eRttUpdate(
+                participant, e2eRtt));
     },
 
     /**
      * Add a subscriber to be notified when stats are updated for a specified
      * user id.
      *
      * @param {string} id - The user id whose stats updates are of interest.
-     * @param {Function} callback - The function to invoke when stats for the
-     * user have been updated.
+     * @param {Function} statsCallback - The function to invoke when stats for
+     * the user have been updated.
+     * @param {Function} e2eRttCallback - The function to invoke when the
+     * end-to-end RTT is updated.
      * @returns {void}
      */
-    subscribeToClientStats(id: ?string, callback: Function) {
+    subscribeToClientStats(
+            id: ?string, statsCallback: Function, e2eRttCallback: Function) {
```

Comment:
Because e2e rtt updates at a different interval as other stats?

Reply:
> Because e2e rtt updates at a different interval as other stats?

I extended the same callback to take more arguments (stats, e2eRtt, region).

Reply:
Would it be difficult to take it further and make the callback itself be able to handle the stats object having e2eRtt and region instead of accessing them as separate arguments?